backend/tracim_backend/views/core_api/schemas.py

```diff
@@ -109,6 +109,22 @@ def _deserialize(self, value, attr, data, **kwargs):
         return None
 
 
+class PositiveIntegerListField(StrippedString):
+    def _deserialize(self, value, attr, data, **kwargs):
+        value = super()._deserialize(value, attr, data, **kwargs)
+        if value:
+            return [int(item) for item in value.split(",")]
+        return None
```

Comment:
You should add a specific validator for this one to avoid weird None case, see `regex_string_as_list_of_int`

Also, we should probably avoid code replication here, we currently do something very similar to this with `string_to_list` method

Reply:
done